src/app/ui/members-quest-approval/members-quest-approval.component.ts

```diff
@@ -35,13 +37,19 @@ export class MembersQuestApprovalComponent implements OnInit {
     private dialog: MatDialog,
     private notify: NotifyService,
     private playerQuest: PlayerQuestService,
-    private season: SeasonService) {}
+    private season: SeasonService,
+    private teamsService: TeamsService) {}
 
   async ngOnInit() {
     if (this.user && this.user.membership)  {
-      const teamId = this.user.membership.teamId;
-      const seasonId = await this.season.getEnabledSeasonId().toPromise();
-      this.questsForApproval$ = this.playerQuest.getAllMemberSubmittedQuests(seasonId, teamId);
+      this.seasonId = await this.season.getEnabledSeasonId().toPromise();
```

Comment:
lets just use the seasonId from auth.service, we don't really need it everytime this component is initialize.
```suggestion
      this.seasonId = this.auth.seasonId;
```

Reply:
removed the seasonservice and used the auth service instead